packages/sdk/src/modules/did/index.ts

```diff
@@ -152,24 +162,44 @@ export class Did extends Base {
     return `did:peaq:${address}`;
   }
 
-  private _createVerificationMethod(address: Address) {
+  private _createVerificationMethod(verification: Verification, address: Address) {
     const id = uuidv4();
```

Comment:
The format of the verification `id` is incorrect.  UUID shouldn't be used here.
Please check this [example 13](https://www.w3.org/TR/did-core/#example-various-verification-method-types) and [example 14](https://www.w3.org/TR/did-core/#example-various-verification-method-types) of the did core specification.

Reply:
@irediaes

Using the cmd: 
`await did.create({ name: "alice_test_4789", customDocumentFields: {verifications: [{type: peaqDidProto.VerificationType.SR25519VERIFICATIONKEY2020}]} });`

Now creates the corresponding did document `      {
        id: 'did:peaq:5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY',
        controller: 'did:peaq:5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY',
        verificationmethodsList: [
          {
            id: 'did:peaq:5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY#keys-1',
            type: 1,
            controller: 'did:peaq:5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY',
            publickeymultibase: 'd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d'
          }
        ],
        signature: undefined,
        servicesList: [],
        authenticationsList: [
          'did:peaq:5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY#keys-1'
        ]
      }`